ansible/grafana-dashboards/idr-vertical.json

```diff
@@ -428,12 +428,10 @@
       {
         "allValue": null,
         "current": {
-          "tags": [],
-          "text": "omero + database + docker",
+          "text": "omero + database",
           "value": [
             "omero",
-            "database",
-            "docker"
```

Comment:
Where has docker gone?

Reply:
It's been gone for years, ever since we separated the deployment of the VAE from IDR.

Reply:
But isn't it coming back with micro services? (and idr-ftp?)

Reply:
These are used for selecting a subset of hostnames, it's not actually anything to do with what's running on the system. We don't have nodes named `.*docker.*`, the current microservice works run the docker micoservices on the `omero*` servers since some of them require filesystem access anyway.